### PR TITLE
fix: explicitly add python2.7 requirement for manta wrapper to avoid python 1.6 being pulled

### PR DESCRIPTION
**Pull request by ipstone**
### Description

The current manta wrapper will result in pulling python 1.6 into the conda environment when using --use-conda for snakemake runs.

This update just added `python=2.7` requirement to the manta wrapper requirement, which fixed the issue on my system.

Adding python2.7 requirement for manta wrapper fixed the problem of solver would pull python 1.6 when using manta wrapper.

### CODE REVIEW

**Comment by ipstone** [comment]
@fgvieira based on these source codes: https://github.com/Illumina/manta/blob/master/src/python/bin/configManta.py

it seems that manta only supports python2 for the scripts setup/calling the binaries. so I guess python=2.7 is safe, though I didn't specifically test if python3 works.

File: bio/manta/environment.yaml [hunk 1]
```diff
@@ -3,5 +3,6 @@ channels:
   - bioconda
   - nodefaults
 dependencies:
+  - python =2.7
```
Comment (fgvieira) [inline bio/manta/environment.yaml:6]:
```suggestion
  - python =2.7.15
```

File: bio/manta/environment.yaml [hunk 2]
```diff
@@ -3,5 +3,6 @@ channels:
...
   - manta =1.6.0
   - bcftools =1.17
```
Comment (fgvieira) [inline bio/manta/environment.yaml:8]:
```suggestion
  - bcftools =1.18
```

**Comment by fgvieira** [comment]
Maybe the best way would be to create a new `environment.linux-64.pin.txt` file pinning all packages in the environment.

**Pull request closed without merging**